named_entity_recognition/README.md

```diff
@@ -0,0 +1,35 @@
+# Named Entity Recognition with Kubeflow and Keras 
+
+This example demonstrates how you can use Kubeflow to train and deploy a Keras model with a custom prediction routine. 
```

Comment:
Perhaps call out that training runs in-cluster and deployment on AI Platform.

Reply:
It would also be useful to point out somewhere that you are using reusable pipelines components

Reply:
✓ done